### ISSUE TITLE
shortcut "?" not working

### ISSUE THREAD

**Issue by aisbergde** [benrhughes/todotxt.net#407]
I am using a German keyboard layout
The important shortcut "?" to get the help windows doesn't work
other shortcuts work, but I did not check all of them

**Comment by daveliao** [comment]
@aisbergde what's the keyboard you have installed, QWERTZ? I want to try changing my keyboard layout and see if I can replicate your issue...

**Comment by aisbergde** [comment]
yes, the standard german keyboard is qwertz. But I think there could be different qwertz keyboards. "?" is on a specific place. 

[German keyboard layout - Wikipedia](https://en.m.wikipedia.org/wiki/German_keyboard_layout)

**Comment by daveliao** [comment]
@aisbergde - I'm able to reproduce your issue. I switched my keyboard from "English (United States) QWERTY" to "German (QWERTZ)".

Looking into todotxt.net's code, I see the app listens for _key="OemQuestion" Modifiers="Shift"_. OemQuestion is the standard question mark key _on US keyboards_, and the modifier "Shift" means the SHIFT key (either one) must be pressed to trigger the About window.

The easiest solution (for a developer/contributor to implement) would be to change the "?" shortcut to something more global/standard, like "F1". I tested F1 myself, and it worked fine for me...

If there are any experienced Windows developers out there, they can probably figure out a more universal way to represent "?" without using OemQuestion... I researched this for a little while today but there's no obvious answer to me.

**Comment by daveliao** [comment]
@benrhughes - hi Ben! Thoughts on changing the "?" shortcut to "F1"? In Chrome, Word, and other major apps, F1 typically opens the help page for an app.

**Comment by chipbite** [comment]
Confirmed, does not work for swedish layout either. That said:
Seems rather very low prio to me, IMO.
